src/engine/OrderBy.cpp

```diff
@@ -80,17 +80,17 @@ void OrderBy::computeResult(ResultTable* result) {
   // TODO<joka921> proper timeout for sorting operations
   double remainingSecs =
       static_cast<double>(_timeoutTimer->wlock()->remainingMicroseconds()) /
-      1000000;
+      1'000'000;
   if (getExecutionContext()
-              ->getSortPerformanceEstimator()
-              .EstimateSortTimeInSeconds(subRes->size(), subRes->width()) /
-          SORT_ESTIMATE_CANCELLATION_FACTOR >
-      remainingSecs) {
+          ->getSortPerformanceEstimator()
+          .estimateSortTimeInSeconds(subRes->size(), subRes->width()) >
```

Comment:
Since the function returns the time, it is better called `estimatedSortTimeInSeconds`

That also reads better in the code where it is used, like here